src/olympia/amo/celery.py

```diff
@@ -50,13 +53,21 @@ def _serialize_args_and_kwargs_for_eager_mode(
             args, kwargs = serialization.loads(data, content_type, content_encoding)
         return args, kwargs
 
+    def original_apply_async(self, args=None, kwargs=None, **extrakw):
```

Comment:
Can you add a comment why it would be necessary to define this method? What is the use case?

Reply:
If the only use case is to call it within the apply_async method, can we not directly call super().apply_async() ? Is it possible to make this method private?

Reply:
This method used to be provided by django-post-request-task in case you wanted to bypass, well, django-post-request-task. It's used once elsewhere in our code to trigger a task immediately without waiting for the transaction on purpose. I'll document it.

Reply:
0d6dd7911864e6e2590592b5c24d770c0de195d0